### PR TITLE
Add recipe for Bottle

### PR DESCRIPTION
**Pull request by hajapy**
Adds a recipe for [Bottle](http://bottlepy.org/).

### CODE REVIEW

File: recipes/bottle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "0.12.9" %}
+{% set sha256 = "45285ad084ca054d821ceef8bd95462efd38e13ecbef13a82c22b6472a6f4b2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bottlepy/bottle/archive/{{ version  }}.tar.gz
```
Comment (goanpeca) [inline recipes/bottle/meta.yaml]: Is it better to use pypi @jakirkham? What is the idea?

Where are all these "suggested guidelines" being collected?

Comment (jakirkham) [inline recipes/bottle/meta.yaml]: We had this discussion awhile back. Some people wanted to build directly from source. Other preferred going with a package index. After a fair bit of discussion, we decided to let both continue. The only caveats are that we must download some kind of compressed file (no VCS clones) and if it comes from source it must be the original source (no forks).

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
-{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
-# sha256 is the prefered checksum -- you can get it for a file with:
-#  `openssl sha256 <file name>`.
-# You may need the openssl package, available on conda-forge
-#  `conda install openssl -c conda-forge``
-
```
Comment (jakirkham) [inline recipes/example/meta.yaml]: Please restore the `example`.

File: recipes/bottle/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    Bottle is a fast, simple and lightweight WSGI micro web-framework for Python. 
+    It is distributed as a single file module and has no dependencies other than the Python Standard Library.
+  doc_url: http://bottlepy.org/docs/0.12/
+  dev_url: https://github.com/bottlepy/bottle
+
+extra:
+  recipe-maintainers:
+    - hajapy
+
+
```
Comment (jakirkham) [inline recipes/bottle/meta.yaml]: Please have only one terminal newline.

File: recipes/bottle/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bottlepy/bottle/archive/{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
```
Comment (jakirkham) [inline recipes/bottle/meta.yaml]: No need for `--record record.txt` if we are not using `setuptools`.

File: recipes/bottle/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/bottle/meta.yaml]: As for the other part of your question, we have these [guidelines]( https://github.com/conda-forge/conda-forge.github.io/blob/1c12a1d6f82064c43575997138c6990c031eb9cc/docs/guidelines.rst ). Though they don't mention this point. It came up in a meeting instead. Perhaps it should be added.

**Comment by hajapy** [comment]
Thanks @jakirkham @goanpeca for the feedback. Changes made. Is there anything else to be done?

**Comment by goanpeca** [comment]
:+1:

File: recipes/bottle/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  description: |
+    Bottle is a fast, simple and lightweight WSGI micro web-framework for Python. 
+    It is distributed as a single file module and has no dependencies other than the Python Standard Library.
+  doc_url: http://bottlepy.org/docs/0.12/
+  dev_url: https://github.com/bottlepy/bottle
+
+extra:
+  recipe-maintainers:
+    - hajapy
+
```
Comment (jakirkham) [inline recipes/bottle/meta.yaml]: nit: please drop extra newline.

**Comment by jakirkham** [comment]
Minor nit. Otherwise LGTM.

**Comment by hajapy** [comment]
Fixed minor nit.

**Comment by jakirkham** [comment]
Thanks @hajapy.

**Pull request merged**